src/query/interpreter.cpp

```diff
@@ -1897,7 +1908,7 @@ PreparedQuery PrepareTriggerQuery(ParsedQuery parsed_query, bool in_explicit_tra
       case TriggerQuery::Action::CREATE_TRIGGER:
         trigger_notification.emplace(SeverityLevel::INFO, NotificationCode::CREATE_TRIGGER,
                                      fmt::format("Created trigger {}.", trigger_query->trigger_name_));
-        EventCounter::IncrementCounter(EventCounter::TriggersCreated);
+        Statistics::IncrementCounter(Statistics::TriggersCreated);
```

Comment:
I think this is not best place as trigger can exist and this is then incremented although trigger was not created